dlpy/applications/__init__.py

```diff
@@ -26,4 +26,5 @@
 from .shufflenet import ShuffleNetV1
 from .inception import InceptionV3
 from .rcnn import Faster_RCNN
-from .unet import UNet
\ No newline at end of file
+from .unet import UNet
+from .efficientnet import *
```

Comment:
you dont need to import all, please just add the actual api that users will use here

Reply:
done